src/main/kotlin/racingcar/domain/Race.kt

```diff
@@ -0,0 +1,25 @@
+package racingcar.domain
+
+class Race {
+    private val cars: List<Car>
+
+    constructor(carCount: Int) {
+        this.cars = (0 until carCount).map { Car() }
+    }
+
+    constructor(cars: List<Car>) {
+        this.cars = cars
+    }
+
+    fun run() {
+        for (car in cars) car.move()
```

Comment:
코틀린 컬렉션의 함수를 사용하면 어떨까요?

```suggestion
        cars.forEach { it.move() }
```